app/charts/bar/bars-state-props.ts

```diff
@@ -114,18 +114,18 @@ export const useBarsStateData = (
   variables: BarsStateVariables
 ): ChartStateData => {
   const { chartConfig, observations } = chartProps;
-  const { sortData, xDimension, getXAsDate, getY, getTimeRangeDate } =
+  const { sortData, yDimension, getYAsDate, getX, getTimeRangeDate } =
     variables;
   const plottableData = usePlottableData(observations, {
-    getY,
+    getX,
   });
   const sortedPlottableData = useMemo(() => {
     return sortData(plottableData);
   }, [sortData, plottableData]);
-  const data = useChartData(sortedPlottableData, {
+  const data = useBarChartData(sortedPlottableData, {
```

Comment:
I think we still should be able to `useChartData`, as this is generic for every chart type 🤔